README.md

```diff
@@ -1,3 +1,21 @@
-# ILITEK ILI9163C Sensor Driver
+# ILITEK ILI9163C display Driver
 
-ILITEK ILI9163C display controller sensor driver for Zephyr OS.
+ILITEK ILI9163C display driver for Zephyr OS.
+
+## Hardware requirements
+- Zest_Display_LCD
+- Any Zest Core board
+
+## Usage
+This dispalay driver can be used to display and draw text, images, and shapes in highly readable form.
+
+### Build
+Use the Zest_Display_LCD shield:
+```bash
+west build -b <board> samples -- -DSHIELD=zest_display_lcd
```

Comment:
Is it correct to ask in the driver to use a shield?

Reply:
Resolved in 1ea59ed680dc881d3afea2fa9cafe1a5cf060ca0.